.circleci/config.yml

```diff
@@ -102,12 +102,6 @@ workflows:
   version: 2
   test:
     jobs:
-      - test_node10:
```

Comment:
Removing the circleci files so it doesn't fail and notify everyone who's listening lol 
I'll add an issue to redo this with github actions.